### PR TITLE
Add function bool Well::hasCompletion( int )

### PR DESCRIPTION
**Pull request by joakim-hove**
Utilities in order to support completion summary keywords like: `WOPRL`

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp [hunk 1]
```diff
@@ -546,7 +547,7 @@ class Well {
       keyword.
     */
     std::map<int, std::vector<Connection>> getCompletions() const;
-
+    bool hasCompletion(int completion) const;
```
Comment (atgeirr) [inline opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp:550]: Looks ok, but I wonder what that `int completion` is. I think it is an "identifier" and not an index, I guess what I'm really looking for is a way to discover (comment for example) what number I am supposed to plug in here.

Comment (joakim-hove) [reply opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp:555]: Comment added.

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**